src/Mod/Part/Gui/TaskAttacher.cpp

```diff
@@ -951,25 +951,27 @@ void TaskAttacher::visibilityAutomation(bool opening_not_closing)
                 editObj = editVp->getObject();
         }
         try{
+            ObjectName = ViewProvider->getObject()->getNameInDocument();
             QString code = QString::fromLatin1(
                 "import Show\n"
-                "tv = Show.TempoVis(App.ActiveDocument, tag= 'PartGui::TaskAttacher')\n"
+                "_tv_%4 = Show.TempoVis(App.ActiveDocument, tag= 'PartGui::TaskAttacher')\n"

```

Comment:
You've just renamed a variable. Probably, there was a name collision. Can you give a clue on how to invoke that collision?

Reply:
If my recollection is correct, it is because `tv` is also used by ViewProviderSketch. If the user invoke sketch edit before closing task attercher, `tv` will be gone.